razzle.config.js

```diff
@@ -8,15 +11,29 @@ module.exports = {
         A prod build will fail if the API changes so it is fairly safe.
       */
       appConfig.module.rules.shift();
-      // This is to pass the bundlde performance test in CI
-      appConfig.performance = Object.assign(
-        {},
-        {
-          maxAssetSize: 350000,
-          maxEntrypointSize: 350000,
-        },
-      );
+      // Setup bundle analyser
+      if (target === 'web') {
+        appConfig.plugins.push(
+          new BundleAnalyzerPlugin({
+            analyzerMode: 'static',
+            defaultSizes: 'gzip',
+            generateStatsFile: true,
+            openAnalyzer: false,
+            reportFilename: '../../reports/webpackBundleReport.html',
+            statsFilename: '../../reports/webpackBundleReport.json',
+          }),
+        );
+      }
     }
+
+    // This is to override bundle performance test
+    appConfig.performance = Object.assign(
```

Comment:
There is little value in cloning objects here. You could just explicitly assign an object to `performance`.